documentation/jetty/modules/operations-guide/pages/tools/index.adoc

```diff
@@ -29,11 +29,16 @@ Username: <1>
 Password: secret <2>
 OBF:1yta1t331v8w1v9q1t331ytc <3>
 MD5:5eBe2294EcD0E0F08eAb7690D2A6Ee69 <4>
+...
+MD:SHA-1:E5E9Fa1bA31eCd1aE84f75CaAa474f3a663f05F4
```

Comment:
OBF can be reversed to a password.
MD5, SHA1-, SHA-256 is 1 direction, it cannot be reversed into a password.

Is that nuance worth mentioning?

Reply:
It's almost like describing a stored password as verifying an incoming password (all storage types supported) or obtaining an outgoing password (only reversible supported)